core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackService.java

```diff
@@ -371,6 +388,10 @@ public void onLoadChildren(@NonNull String parentId,
         if (parentId.equals(getResources().getString(R.string.app_name))) {
             // Root List
             mediaItems.add(createBrowsableMediaItemForRoot());
```

Comment:
I'm thinking replacing this line with the following may do the trick:
`
try {
    if (!(taskManager.getQueue().isEmpty())) {
        mediaItems.add(createBrowsableMediaItemForRoot());
    }
} catch (InterruptedException e) {
    e.printStackTrace();
}`